### DISCUSSION TITLE
Which secondary animation app works best with toonsquid?

### DISCUSSION THREAD

**Discussion by ArtsyStar1** [keiwando/toonsquid#705] [Q&A] [answered]
As soon as I switch iPads, I will start animating with toonsquid more frequently because I can’t use adobe animate all the time(and because it is currently not available on iPad). 
But I’m not sure if I want to make an entire animation on toonsquid, so I wonder if using another animation app to help me out will be useful, plus I like to animate in the south park art style, which means I’m using a digital cutout animation format.
The animation apps I want to use alongside toonsquid are either callipeg or procreate dreams, which one should I use?

**Comment by Cat-Butler** [comment, marked as answer]
What particular effects/features do you require in the other app that are not in ToonSquid? That might help define your answer. 

Both your suggestions are more focused on frame-by-frame drawing (albeit with some level of transformation keyframing), so are potentially less useful for cutout/South Park style animation. It is possible to do hierarchy-style cutout animation in Dreams by abusing timeline layer grouping, but it is clumsier than ToonSquid's hierarchy feature.
Note: Dreams at the moment also seems to work better if you combine it with Procreate the paint app rather than using it standalone.

Other companion apps to consider could be art apps (vector and/or raster) for backgrounds and props, an audio app for recording/editing voice and other sound, and/or a video app for compositing/editing/FX (depending on your needs, this might also help for secondary animation keyframing).

**Comment by brindacier** [comment]
ToonSquid is very powerful but you could use LumaFusion to edit the created movie with more transitions between scenes.
Also there is no way to work with alpha channel and green screen in ToonSquid. So, you can’t import a movie and get rid of the background to merge it with your animation so LumaFusion is the solution.
Frankly, I used to animate with flash and later on with procreate ( witch I still use to draw my characters and animate shorts frame by frame clips) but now I will stick with ToonSquid ! I really love the app and honestly, I bought Dreams but I don’t like at all its interface and ToonSquid is more useful for my needs.
